docs/openapi.yaml

```diff
@@ -632,3 +632,191 @@ paths:
+            content:
+              application/json:
+                schema:
+                  type: object
+                  example:
+                    - {
+                      "Data Type": {
+                        "total": 275,
+                        "names": [
+                          {
+                            "Genotype": 103,
+                            "Clinical Phenotype": 100,
+                            "DCC Harmonized": 24,
+                            "WGS": 20,
+                            "SNP/CNV Genotypes (NGS)": 6,
+                            "RNA-Seq": 5,
+                            "WXS": 5,
+                            "Targeted-Capture": 3,
+                            "miRNA-Seq": 3,
+                            "CNV Genotypes": 2
+                          }
+                        ]
+                      }
+                     }
+      summary: Get tag counts information
+      tags:
+      - Aggregate
+  /aggregate/metadata:
+    get:
+      description: "Returns a list of all commons with data in the aggregate metadata-service"
+      operationId: get_aggregate_metadata
+      parameters:
+      - in: query
+        name: limit
+        required: false
+        schema:
+          title: limit
+          type: integer
+          default: 20
+      - in: query
+        name: offset
+        schema:
+          title: offset
+          type: integer
+          default: 0
+        description: "Return results at this given offset"
+      - in: query
+        name: flatten
+        schema:
+          title: flatten
+          type: boolean
+          default: false
+        description: "Return the results without grouping items by commons"
+      - in: query
+        name: pagination
+        schema:
+          title: pagination
+          type: boolean
+          default: false
+        description: "If true will return a pagination object in the response"
+      - in: query
+        name: counts
+        schema:
+          title: counts
+          type: string
+          default: ""
+        description: "Return count of a field instead of the value if field is an array\
+          \ otherwise field is unchanged. If field is null will set field to 0.\
+          \ Multiple fields can be compressed by comma separating the field names:\
+          \ _files,authors"
+      responses:
+        '200':
+            description: Successful Response
+            content:
+              application/json:
+                schema:
+                  type: object
+                  additionalProperties: true
+                  example:
+                    - {
+                      "commonA" : [ { id2: { name: "bear" } } , { id3: { name: "cat" } } ],
+                      "commonB" : [ { id200: { name: "shark" } } , { id312: { name: "bass" }} ]
+                      }
+      summary: Get metadata records from aggregate metadata
+      tags:
+      - Aggregate
+  /aggregate/metadata/{name}:
```

Comment:
the `name` param should be described in the endpoint parameters

Reply:
added

Reply:
Added